### PR TITLE
DEL Init mailer module

### PR DESCRIPTION
**Pull request by freeman91**
Story: [65](https://5911-capstone.atlassian.net/browse/OCA-65)

## Summary

- Init mailer module
- Init invite email template

## Screenshot(s)

Screen Shot 2020-11-14 at 3 11 43 PM

### CODE REVIEW

File: api/mailer/__init__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      token: token created for the email
+      email: address the email is being sent to
+
+      Returns:
+        None
+    """
+
+    mail = Mail(app)
+    with open("api/mailer/templates/invite_template.txt", "r") as f:
+        template = Template(f.read())
```
Comment (NickHackman) [inline api/mailer/__init__.py:29]: Isn't there a way to serve up a jinja template rather than serving a rendering text file?

**Review by NickHackman — APPROVED** [review]
LGTM, not sure why my changes (from the culture-view branch) are needed for this update, but sure. 60 commits makes reviewing PRs a little difficult especially with a bunch of stuff that doesn't really pertain to the change.

**Comment by freeman91** [comment]
@NickHackman All of those commits aren't need I'll make a new pr and cherry-pick the commits I want.
Same thing happened with my other pr. I merged culture-view just to how you guys were doing that. I'll make new PRs

File: api/mailer/__init__.py (hunk 1 above)
Comment (freeman91) [reply api/mailer/__init__.py:29]: I'll look into the template

**Pull request closed without merging**